lib/slack_message/error_handling.rb

```diff
@@ -7,8 +7,10 @@ def self.raise_post_response_errors(response, params, profile)
     body  = JSON.parse(response.body)
     error = body.fetch("error", "")
 
-    if ["invalid_blocks", "invalid_blocks_format"].include?(error)
-      raise SlackMessage::ApiError, "Couldn't send Slack message because the serialized message had an invalid format"
+    if error == "invalid_blocks"
+      raise SlackMessage::ApiError, "Couldn't send Slack message because the request contained invalid blocks:\n#{JSON.pretty_generate(params[:blocks])}"
+    elsif error == "invalid_blocks_format"
+      raise SlackMessage::ApiError, "Couldn't send Slack message because blocks is not a valid JSON object or doesn't match the Block Kit syntax:\n#{JSON.pretty_generate(params[:blocks])}"
```

Comment:
I'm concerned that this output would be pretty unfriendly, so including it in the default output is a bit gnarly. Similarly, I think this would make Rollbar have a heart attack. How would you feel about moving it to respect `SlackMessage::Configuration.debugging?` and then changing the message? Something like:

```
message = if SlackMessage::Configuration.debugging?
  JSON.pretty_generate(params[:blocks])
else
  "Enable debugging in configuration to view block data."
end
```

Reply:
I think that's a great idea!